src/main/resources/db/migration/V20200526141500__lowercase_email_username.sql

```diff
@@ -0,0 +1,5 @@
+
+update users_data set
```

Comment:
Why username should be in a lower case? I guess there are authentication providers that might be case-sensitive.